spec/classes/sensu_package_spec.rb

```diff
@@ -294,7 +294,7 @@
         let(:params) { { :purge => 'a_string' } }
 
         it 'should fail' do
-          expect { should create_class('sensu') }.to raise_error(/not a Hash/)
+          expect { should create_class('sensu') }.to raise_error(Puppet::PreformattedError)
```

Comment:
```Puppet::PreformattedError``` will accept any error thrown, independently from the reason for the fail. The intention here is to ensure that the module really fails on the validation errors.